### PR TITLE
thierry's-portfolio-continuation

### PR DESCRIPTION
**Pull request by axcell-0**
this milestone was created using:

- Index.html file
- Style.css file
- Linters
- Readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +47,183 @@ <h1>THIERRY <br> NOUMBUH</h1>
             <img src="assets/" alt="">
           </div>
         </section>
+
+        <section class="project-section-container">
+            <div class="project-section">
+                <h2>projects</h2>
```
Comment (joel-kalema) [inline index.html:53]: Keep doing great work @axcell-0 👍, but let's make some changes to stick to the design 💪   

- [ ] Kindly make sure the `languages section` look like the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃. Please, fix the `margin`, `font`, and `padding` 💪 .

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-06-01 155956| |Screenshot 2023-06-01 160149| |

File: index.html [hunk 2]
```diff
@@ -47,6 +47,183 @@ <h1>THIERRY <br> NOUMBUH</h1>
...
+                            <button class="project-button">
+                            <span>See this project</span> 
+                                <img src="assets/images/arrow.svg" alt="arrow" class="arrow">
+                            </button>
+                        </div>
+                    </section>
+
+                        <section class="containing-6">
+                            <div class="rectangle_21">
+                                <img src="assets/images/rectangle_25.svg" alt="rectangle_21" />
```
Comment (joel-kalema) [inline index.html:139]: - [ ] Also, please, make sure your projects images don't have `margin` or `padding` to stick to the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

 ### Exemple

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-06-01 161622| |Screenshot 2023-06-01 161600| |


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-06-01 161750| |Screenshot 2023-06-01 161718| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @axcell-0 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/style.css [hunk 3]
```diff
@@ -94,3 +94,120 @@ h1 {
...
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  color: #1a2236;
+  margin: 39px 16px;
+}
+
+.rectangle_21 img {
+  width: 100%;
+}
```
Comment (zieeco) [inline assets/style.css:115]: - Hey, great job making the changes requested by the previous reviewer, however, the issue with the slight spaces on the images [in this review summary](https://github.com/axcell-0/Thierrys-portfolio/pull/2#discussion_r1213239261). To improve on this to not make it obvious, kindly add a display property of the block and your design will be better.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @axcell-0,





Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice design. ✔️
- Descriptive PR. ✔️
- All linters are passing heavy_check_mark
- Good commit history heavy_check_mark
- Follow GitHub flow heavy_check_mark

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @axcell-0 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Used CSS `display: block;` property ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @axcell-0 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Used CSS `display: block;` property ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**